curriculum/challenges/english/10-coding-interview-prep/rosetta-code/state-name-puzzle.md

```diff
@@ -15,7 +15,7 @@ dashedName: state-name-puzzle
 
 Write a function to solve the challenge for a given array of actual U.S. state names, and for fictional state names.
 
-The function should return an array. Each element should be an object in this form: `{"from":[],"to":[]}`. The `"from"` array should contain the original names and the `"to"` array should contain the resultant names.
+The function should return an array. Each element should be an object in this form: `{from:[],to:[]}`. The `from` array should contain the original names and the `to` array should contain the resultant names.
```

Comment:
The quotes here match how the object would print out in a console.log

Reply:
Yeah. I agree with you here.

Reply:
Here, I don't mind the quotes in the object - `{ "from": [], "to": [] }` <- note that it could probably use some spaces. I would not use quotes around the `from` and `to` in the next sentence.